pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -51,7 +52,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/labels"
+	k8sApiLables "k8s.io/apimachinery/pkg/labels"
```

Comment:
I renamed because, `labels` is freely used as a variable name and it is also imported in the same scope. Do you think it is ok to keep it as `labels` ? (it failed a style check in my IDE though. )

Reply:
I don't feel strongly either way, maybe `k8slabels`